.depcheckrc

```diff
@@ -0,0 +1,7 @@
+ignores: [
+  "@types/config",
+  "@types/sinon",
+  "concurrently",
+  "nodemon",
+  "sinon"
```

Comment:
some of these packages aren't used anywhere, like sinon shouldn't be an installed package?

Reply:
removed. This was a result of not having a complete unit test coverage. Added sinon for mocking/stubbing and etc.

Reply:
You can remove it from `.depcheckrc` as well.

Reply:
What about the other ignored deps here?  Can they be uninstalled as well?

Reply:
For devDependencies like that, its fine.

Reply:
likely should be the same ignore list as https://github.com/digicatapult/morello-api/blob/main/.depcheckrc + `pg`?